src/cephadm/cephadm.py

```diff
@@ -55,6 +55,7 @@
 DEFAULT_GRAFANA_IMAGE = 'quay.io/ceph/ceph-grafana:9.4.7'
 DEFAULT_HAPROXY_IMAGE = 'quay.io/ceph/haproxy:2.3'
 DEFAULT_KEEPALIVED_IMAGE = 'quay.io/ceph/keepalived:2.2.4'
+DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:latest'  # TODO use a the tag instead of latest
```

Comment:
Right now this is:
```suggestion
DEFAULT_NVMEOF_IMAGE = 'quay.io/ceph/nvmeof:0.0.1'  # TODO use a the tag instead of latest
```

Ideally this should end up with a pseudo-stable release, like `1.y.z`, as `0.y.z` is for [dev releases](https://semver.org/).

Reply:
fixed